ios/GatewayManager.m

```diff
@@ -66,7 +66,7 @@ - (void)configureStripeGateway:(NSDictionary *_Nonnull)gatewayParameters
 }
 
 
-- (void)savePaymentMethod:(NSString *)secret cardParams:(NSDictionary *)cardParams completion:(void (^)(NSString * _Nullable, NSError * _Nullable))completion
+- (void)savePaymentMethod:(NSString *)clientSecret cardParams:(NSDictionary *)cardParams completion:(void (^)(NSString * _Nullable, NSError * _Nullable))completion
```

Comment:
Sorry I'm clear now, in RNP.m call it `savePaymentMethod`, in `GatewayManager` call it the same as Stripe `confirmSetupIntent`